client/app/components/datagrid/Note.tsx

```diff
@@ -0,0 +1,17 @@
+export const registrationRequestNote =
+  "Once “Approved”, the operation will be registered for the B.C. Output-Based Pricing System and exempted from the carbon tax.";
+
+interface Props {
+  message: string;
+}
+
+const Note = ({ message }: Props) => {
+  return (
+    <>
+      <strong>Note: </strong>
+      {message}
+    </>
```

Comment:
Nice job, I think this PR is almost good to go though I think we could consider wrapping this in a div (likely need to use a flex box on it and the text might need to be wrapped as well). 

This works well on the operations page though using just a fragment leaves it at the mercy of the `display`/styling of the parent component and could cause issues in the future.